frontend/src/components/controls/form-controls/select/SimpleCreatableSelectWithOptions.tsx

```diff
@@ -0,0 +1,35 @@
+import Creatable from 'react-select/creatable';
+import React from 'react';
+import {SelectItem} from '../../../../models';
+import {t} from '../../../utils';
+
+export type Option = {
+  label: string,
+  value: string
+}
+
+export type SimpleCreatableSelectWithOptionsProps = {
+  options: Option[];
+  value: string;
+  onChange: Function;
+  isClearable?: boolean;
+};
+
+
+export const SimpleCreatableSelectWithOptions = ({options, value, onChange, isClearable = false, ...props}: SimpleCreatableSelectWithOptionsProps) => {
```

Comment:
Wat doet deze anders dan de al bestaande CreatableSelects?

Reply:
De bestaande select gaat de value in de options mappen naar het label. Om de bestaande code zo min mogelijk te impacteren, heb ik een nieuw component gemaakt waarbij er van wordt uitgegaan dat de options aparte labels bevat.